From: Antonio Radici <antonio@debian.org>
Date: Thu, 27 Feb 2014 16:17:42 +0100
Subject: dont_document_not_present_features

As the patch says, this will add an include so only
documented options for Debian will be used.

To apply to neomutt by:
From 43c380811ed3f8b1d163fec5d226a1bf2505e8ab Mon Sep 17 00:00:00 2001
From: Richard Russon <rich@flatcap.org>
Date: Fri, 6 May 2016 00:23:59 +0100
Subject: [PATCH 08/22] debian-specific/dont_document_not_present_features

---
 functions.h | 2 +-
 init.h      | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

Index: neomutt-20160818/functions.h
===================================================================
--- neomutt-20160818.orig/functions.h
+++ neomutt-20160818/functions.h
@@ -32,7 +32,7 @@
 
 #ifdef _MAKEDOC
 # include "config.h"
-# include "doc/makedoc-defs.h"
+/* # include "doc/makedoc-defs.h" include only documented options for Debian */
 #endif
 
 const struct binding_t OpGeneric[] = { /* map: generic */
Index: neomutt-20160818/init.h
===================================================================
--- neomutt-20160818.orig/init.h
+++ neomutt-20160818/init.h
@@ -19,7 +19,7 @@
 
 #ifdef _MAKEDOC
 # include "config.h"
-# include "doc/makedoc-defs.h"
+/* # include "makedoc-defs.h" include only documented options for Debian */
 #else
 # include "sort.h"
 #endif
